Makefile

```diff
@@ -158,11 +159,15 @@ smokecheck: clean develop
 	./scripts/avocado run passtest.py
 
 check: clean develop check_cyclical modules_boundaries
-	selftests/checkall
+	AVOCADO_CHECK_LEVEL=0 selftests/checkall
```

Comment:
`make check` (aka level 0) is already the default in the sense that the skips default to it.  This only makes it explicit.  With regards to different levels, this is the whole point of this PR, and it comes from the fact that you'd like to keep some tests that are failing on RPM build to still be enabled on Travis.